tests/test_browser.py

```diff
@@ -1593,7 +1593,7 @@ def test_worker(self):
     for file_data in [1, 0]:
       cmd = [EMCC, path_from_root('tests', 'hello_world_worker.cpp'), '-o', 'worker.js'] + (['--preload-file', 'file.dat'] if file_data else [])
       print(cmd)
-      subprocess.check_call(cmd)
+      self.run_process(cmd)
```

Comment:
Why the change?

Reply:
This is kind of an unrelated cleanup.. test code should always run `*self*.run_process` these days so it fails in the right way.